draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1971,15 +2031,28 @@ was also invaluable in achieving coordination with ISO/IEC.
 
 # Some Namespace IDs {#namespaces}
 
-This appendix lists the namespace IDs for some potentially interesting namespaces such those for
+This appendix table, {{namespaceIDs}} the namespace IDs for some potentially interesting namespaces such those for
 {{RFC8499}} domain name system (DNS), {{RFC1738}} uniform resource locators (URLs), {{X660}} object identifiers (OIDs), and {{X500}} distinguished names (DNs).
 
-~~~~ code
-NameSpace_DNS  = "6ba7b810-9dad-11d1-80b4-00c04fd430c8"
-NameSpace_URL  = "6ba7b811-9dad-11d1-80b4-00c04fd430c8"
-NameSpace_OID  = "6ba7b812-9dad-11d1-80b4-00c04fd430c8"
-NameSpace_X500 = "6ba7b814-9dad-11d1-80b4-00c04fd430c8"
-~~~~
+| Namespace | Namespace ID                         |
+| DNS       | 6ba7b810-9dad-11d1-80b4-00c04fd430c8 |
+| URL       | 6ba7b811-9dad-11d1-80b4-00c04fd430c8 |
+| OID       | 6ba7b812-9dad-11d1-80b4-00c04fd430c8 |
+| X500      | 6ba7b814-9dad-11d1-80b4-00c04fd430c8 |
+{: #namespaceIDs title='Namespace IDs'}
+
+Generally speaking, Namespace IDs are allocated as follows:
+
+{: spacing="compact"}
+- The first Namespace ID, for DNS, was calculated from a time-based UUIDv1 and "6ba7b810-9dad-11d1-80b4-00c04fd430c8" used as a starting point.
+- Subsequent Namespace IDs increment the least-significant, right-most bit of time_low "6ba7b810" while freezing the rest of the UUID to "9dad-11d1-80b4-00c04fd430c8".
+- New Namespace IDs MUST use this same logic and MUST NOT use a previously used Namespace ID value.
+- Thus, "6ba7b815" is the next available time_low for a new Namespace ID with the full ID being "6ba7b815-9dad-11d1-80b4-00c04fd430c8".
+- The upper bound for time_low in this special use, namespace ID, is "ffffffff" or "ffffffff-9dad-11d1-80b4-00c04fd430c8" which should be sufficient space for future namespace IDs.
+
+Note that the Namespace ID "6ba7b813-9dad-11d1-80b4-00c04fd430c8" and its usage is not defined by this document or by {{RFC4122}}, as such it MUST NOT be used as a Namespace ID.
+
+New Namespace IDs MUST be documented as per {{IANA}} and {{iana3}}.
```

Comment:
Maybe we can ask Paul Leach more about this origin if this is not done yet. It seems we don't know exactly how they were created and we also don't know what happened with `6ba7b813`. If we know, we can come up with a better way to generate Special UUIDs and eventually even break the current limit.

Reply:
Yeah, I just put what I know and detailed how it is currently. There is nothing I could find in my research on the topic of `6ba7b813` and I saw at least one library using it in some code. So I said "best let sleeping dogs lie"..

I think there has to be some guidance if we are registering such a value but again, I don't think we need to overthink it.